README.md

```diff
@@ -431,6 +431,29 @@ servers:
 | project | `string` | The GCP project name. |
 | topic   | `string` | The topic name.       |
 
+
+#### AWS Glue Server Object
+
+| Field    | Type     | Description                                                |
+|----------|----------|------------------------------------------------------------|
+| type     | `string` | `glue`                                                     |
+| account  | `string` | The AWS account, e.g., `1234-5678-9012`                    |
+| database | `string` | The AWS Glue Catalog database                              |
```

Comment:
Required fields should be marked REQUIRED at the beginning of the description

Reply:
updated